Source/Orts.Simulation/Simulation/Timetables/TTTrain.cs

```diff
@@ -7367,6 +7651,39 @@ public void ProcessTimetableStopCommands(TTTrainCommands thisCommand, int subrou
                     }
                     break;
 
+                case "activate":
+                    if (thisCommand.CommandValues == null || thisCommand.CommandValues.Count < 1)
+                    {
+                        Trace.TraceInformation("No train reference set for train activation, train {0}", Name);
+                        break;
+                    }
+
+                    TriggerActivation thisTrigger = new TriggerActivation();
+
+                    if (plattformReferenceID >= 0)
+                    {
+                        thisTrigger.platformId = plattformReferenceID;
+                        thisTrigger.activationType = TriggerActivationType.StationStop;
+
+                        if (thisCommand.CommandQualifiers != null && thisCommand.CommandQualifiers.Count > 0)
+                        {
+                            TTTrainCommands.TTTrainComQualifiers thisQualifier = thisCommand.CommandQualifiers[0]; // takes only 1 qualifier
+                            if (String.Equals(thisQualifier.QualifierName.Trim().ToLower(), "depart"))
```

Comment:
```suggestion
                            if (thisQualifier.QualifierName.Trim().ToLower() == "depart")
```

Reply:
Oh dear - habbit of a lifetime. I've worked for over 25 years with languages where one could not just compare strings. You found this one but there are many more like this, and I fear there will be more to come because it is just something I will do automatically.

Reply:
That's okay, there's plenty of similar types of not-really-C# coding all through the code that we'll be trying to clean up, when and if we have time. 🙂